Library/Formula/openvdb.rb

```diff
@@ -0,0 +1,100 @@
+  end
+
+  if build.with? "logging"
+    depends_on "log4cplus"
+  end
+
+  resource "test_file" do
+    url "http://www.openvdb.org/download/models/cube.vdb.zip"
+    sha256 "05476e84e91c0214ad7593850e6e7c28f777aa4ff0a1d88d91168a7dd050f922"
+  end
+
+  def install
+    # Adjust hard coded paths in Makefile
+    inreplace "openvdb/Makefile", "/tmp/OpenVDB", "#{prefix}"
+    inreplace "openvdb/Makefile", "BOOST_INCL_DIR := $(HT)/include", "BOOST_INCL_DIR:=#{Formula["boost"].opt_lib}/include"
+    inreplace "openvdb/Makefile", "BOOST_LIB_DIR := $(HDSO)", "BOOST_LIB_DIR:=#{Formula["boost"].opt_lib}"
+    inreplace "openvdb/Makefile", "TBB_INCL_DIR := $(HT)/include", "TBB_INCL_DIR:=#{Formula["tbb"].opt_lib}/include"
+    inreplace "openvdb/Makefile", "TBB_LIB_DIR := $(HDSO)", "TBB_LIB_DIR:=#{Formula["tbb"].opt_lib}/lib"
+    inreplace "openvdb/Makefile", "EXR_INCL_DIR := $(HT)/include", "EXR_INCL_DIR:=#{Formula["openexr"].opt_lib}/include"
+    inreplace "openvdb/Makefile", "EXR_LIB_DIR := $(HDSO)", "EXR_LIB_DIR:=#{Formula["openexr"].opt_lib}/lib"
+    inreplace "openvdb/Makefile", "-lboost_thread", "-lboost_thread-mt"
+
+    inreplace "openvdb/Makefile", "PYTHON_VERSION := 2.6", "PYTHON_VERSION :="
+    inreplace "openvdb/Makefile", "NUMPY_INCL_DIR := /rel/map/generic-2013.22/include", "NUMPY_INCL_DIR :="
+
+  if build.with? "jemalloc"
+    inreplace "openvdb/Makefile", "CONCURRENT_MALLOC_LIB_DIR := $(HDSO)", "CONCURRENT_MALLOC_LIB_DIR:=#{Formula["jemalloc"].opt_lib}/lib"
+  else
+    inreplace "openvdb/Makefile", "-ljemalloc", ""
+  end
+
+  if build.with? "glfw3"
+    inreplace "openvdb/Makefile", "-lglfw", "-lglfw3"
+  else
+    inreplace "openvdb/Makefile", "-lglfw", ""
+  end
+
+  if build.without? "doxygen"
+  inreplace "openvdb/Makefile", "DOXYGEN := doxygen", "DOXYGEN :="
+  end
+
+  if build.with? "cppunit"
+    inreplace "openvdb/Makefile", "CPPUNIT_INCL_DIR := /rel/map/generic-2013.22/sys_include", "CPPUNIT_INCL_DIR:=#{Formula["cppunit"].opt_lib}/include"
+    inreplace "openvdb/Makefile", "CPPUNIT_LIB_DIR := /rel/depot/third_party_build/cppunit/1.10.2-7/opt-ws5-x86_64-gccWS5_64/lib", "CPPUNIT_LIB_DIR:=#{Formula["cppunit"].opt_lib}/lib"
+  else
+    inreplace "openvdb/Makefile", "CPPUNIT_INCL_DIR := /rel/map/generic-2013.22/sys_include", ""
+  end
+
+  if build.with? "log4cplus"
+    inreplace "openvdb/Makefile", "LOG4CPLUS_INCL_DIR := /rel/folio/log4cplus/log4cplus-1.0.3-latest/sys_include", "LOG4CPLUS_INCL_DIR:=#{Formula["log4cplus"].opt_lib}/include"
+    inreplace "openvdb/Makefile", "LOG4CPLUS_LIB_DIR := /rel/folio/log4cplus/log4cplus-1.0.3-latest/library", "LOG4CPLUS_LIB_DIR:=#{Formula["log4cplus"].opt_lib}/lib"
+  else
+    inreplace "openvdb/Makefile", "LOG4CPLUS_INCL_DIR := /rel/folio/log4cplus/log4cplus-1.0.3-latest/sys_include", "LOG4CPLUS_INCL_DIR:="
+  end
+
+  # Blosc is not yet supported.
+  inreplace "openvdb/Makefile", "BLOSC_INCL_DIR := $(HT)/include", "BLOSC_INCL_DIR :="
+
+  ENV.append_to_cflags "-std=c++11"
```

Comment:
This shouldn't be necessary. If it needs `cxx11` please add `needs :cxx11` to the main block and `ENV.cxx11` to `def install`.